cellarium/ml/models/logistic_regression.py

```diff
@@ -104,11 +104,11 @@ def forward(
     def model(self, x_ng: torch.Tensor, y_n: torch.Tensor) -> None:
         W_gc = pyro.sample(
             "W",
-            dist.Laplace(0, self.W_prior_scale).expand([self.n_vars, self.n_categories]).to_event(2),
+            dist.Laplace(0, self.W_prior_scale).expand([self.n_vars, self.n_categories]).to_event(2),  # type: ignore[attr-defined]
```

Comment:
Are these type ignores necessary? The mypy version is currently pinned to 1.10.0 and type checking passed in the previous PR.

Reply:
Well, if it passed before, it should be safe for me to remove these